addons/binding/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/handler/IPBridgeHandler.java

```diff
@@ -64,17 +65,22 @@
 
     private static final String DEFAULT_USER = "lutron";
     private static final String DEFAULT_PASSWORD = "integration";
+    private static final int DEFAULT_RECONNECT = 5;
```

Comment:
That spits out a long rather than an int, but I replaced those two variable names with something more descriptive (DEFAULT_RECONNECT_MINUTES and DEFAULT_HEARTBEAT_MINUTES).